site/src/pages/docs/setup.mdx

```diff
@@ -116,7 +141,6 @@ You might want to use the following build file as a starting point when you set
         }\n
         // Logging
         runtimeOnly 'ch.qos.logback:logback-classic:${versions['ch.qos.logback:logback-classic']}'
-        runtimeOnly 'org.slf4j:log4j-over-slf4j:${versions['org.slf4j:log4j-over-slf4j']}'
```

Comment:
Users do not have to specify slf4j because Armeria core uses it with `api` configuration.

Reply:
Would you also update L131, L133, L137, and L138?

Reply:
Oops, fixed. Thanks!